pkg/nsx/services/vpc/vpc.go

```diff
@@ -679,6 +678,22 @@ func (s *VPCService) GetAVISubnetInfo(vpc model.Vpc) (string, string, error) {
 	return path, cidr, nil
 }
 
+func (s *VPCService) GetNSXLBSNATIP(vpc model.Vpc) (string, error) {
+	log.V(2).Info("Get VPC NSX LB SNAT IP", "VPC", *vpc.Id)
```

Comment:
Get -> Getting? seem the convention uses "Getting" before operation and "Got" after operation finished.

Reply:
Sure, Thanks, fixed.